src/overview/components/Filters.jsx

```diff
@@ -19,12 +23,22 @@ const Filters = ({ showOnlyBookmarks, onShowOnlyBookmarksChange }) => (
                 </span>
             </label>
         </div>
+        {/* TODO: Toggle for Clear Filters button. Need to be modified according to new filters being added. */}
+        {showOnlyBookmarks && (
+            <div
```

Comment:
For better semantics and default behaviour `<button type="button>` would be more appropriate here than `<div>`

Reply:
Will fix that. Just wanted to know that when will the button be visible, when some filters are selected, or all the time and how to implement that?